.travis.yml

```diff
@@ -40,6 +35,19 @@ matrix:
         - sudo ssh default -t 'cd /vagrant && sudo make localrootlessintegration RUNC_USE_SYSTEMD=yes'
         # same setup but with fs2 driver (rootless) instead of systemd
         - sudo ssh default -t 'cd /vagrant && sudo make localrootlessintegration'
+    - name: "centos7"
+      before_install:
+        - sudo ./script/install-vagrant.sh
+        - ln -sf Vagrantfile.centos7 Vagrantfile
+        - sudo vagrant up && sudo mkdir -p /root/.ssh && sudo sh -c "vagrant ssh-config >> /root/.ssh/config"
+      script:
+        # kernel 3.10 (frankenized)
+        - sudo ssh default 'uname -a'
+        - sudo ssh default -t 'sudo -i -- sh -c "cd /vagrant && make localunittest"'
+        - sudo ssh default -t 'sudo -i -- sh -c "cd /vagrant && make localintegration RUNC_USE_SYSTEMD=yes"'
+        - sudo ssh default -t 'sudo -i -- sh -c "cd /vagrant && make localintegration"'
+        - sudo ssh default -t 'sudo -i -- sh -c "cd /vagrant && make localrootlessintegration"'
+
```

Comment:
Maybe create a separate target for these four and use it? Something like `localtest`?

Funny, we already have one, it just needs a variant for systemd :)

Reply:
should be another PR perhaps